Description: non case sensitive url comparison in tests
Forwarded: not-needed
Author: Ondřej Kobližek <koblizeko@gmail.com>

--- a/requests_unixsocket/tests/test_requests_unixsocket.py
+++ b/requests_unixsocket/tests/test_requests_unixsocket.py
@@ -34,7 +34,7 @@
             assert r.headers['X-Requested-Path'] == '/path/to/page'
             assert r.headers['X-Socket-Path'] == usock_thread.usock
             assert isinstance(r.connection, requests_unixsocket.UnixAdapter)
-            assert r.url == url
+            assert r.url.lower() == url.lower()
             if method == 'head':
                 assert r.text == ''
             else:
@@ -62,7 +62,7 @@
             assert r.headers['X-Requested-Query-String'] == 'timestamp=true'
             assert r.headers['X-Socket-Path'] == usock_thread.usock
             assert isinstance(r.connection, requests_unixsocket.UnixAdapter)
-            assert r.url == url
+            assert r.url.lower() == url.lower()
             if method == 'head':
                 assert r.text == ''
             else:
@@ -110,7 +110,7 @@
                 assert r.headers['X-Socket-Path'] == usock_thread.usock
                 assert isinstance(r.connection,
                                   requests_unixsocket.UnixAdapter)
-                assert r.url == url
+                assert r.url.lower() == url.lower()
                 if method == 'head':
                     assert r.text == ''
                 else:
